_data/versions.yml

```diff
@@ -1390,6 +1390,48 @@ v3.1:
       url: https://github.com/projectcalico/routereflector/releases/tag/v0.6.1
 
 v3.0:
+- title: v3.0.12
+  note: ""
+  components:
+    felix:
+      version: v3.0.12
+      url: https://github.com/projectcalico/felix/releases/tag/v3.0.12
+    typha:
+      version: v3.0.12
+      url: https://github.com/projectcalico/typha/releases/tag/v3.0.12
+    calicoctl:
+      version: v3.0.12
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.0.12
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.0.12/calicoctl
+    calico/node:
+      version: v3.0.12
+      url: https://github.com/projectcalico/calico/releases/tag/v3.0.12
```

Comment:
This tag does not exist

Reply:
As mentioned in another PR, the tag won't exist until after the PR is merged.